package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "alfresco-content-app",
-  "version": "4.5.0",
+  "version": "4.4.1",
```

Comment:
I think you should not have that change here

Reply:
removed